tee-worker/omni-executor/executor-storage/src/passkey.rs

```diff
@@ -15,22 +15,31 @@
 // along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
 
 use crate::{Storage, StorageDB};
-use executor_crypto::hashing::{blake2_256, twox_128};
+use executor_crypto::hashing::blake2_256;
 use executor_primitives::AccountId;
 use parity_scale_codec::{Decode, Encode};
 use std::sync::Arc;
 
 const STORAGE_NAME: &str = "passkey_storage";
+const INDEX_STORAGE_NAME: &str = "passkey_account_index";
 
 /// Storage key type: (AccountId, Hash(credential_id))
 pub type PasskeyStorageKey = (AccountId, [u8; 32]);
 
+/// Index storage: Maps AccountId -> Vec<String> (credential_ids)
+/// This enables O(1) lookup of all credential IDs for a given account
+pub type PasskeyAccountIndexKey = AccountId;
+pub type PasskeyAccountIndexValue = Vec<String>;
+
 /// Passkey data structure
 #[derive(Encode, Decode, Clone, Debug, PartialEq, Eq)]
 pub struct PasskeyRecord {
 	pub credential_id: String,
 	pub pubkey: Vec<u8>, // Store SEC1 bytes directly
 	pub created_at: u64,
+	pub omni_account: AccountId,
```

Comment:
Hmm where is this field used?

Reply:
It's not used yet actually. I just added it there in case somewhere needs it. Current map structure doesn't have any directly info about `omni_account`.